### PR TITLE
Jesting Tests completed

### PR DESCRIPTION
**Pull request by Aaron-RN**
Tests written for all functions using Jest module.

### CODE REVIEW

File: src/caesar-cipher.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+const CaesarCipher = (() => {
```
Comment (MauricioRobayo) [inline src/caesar-cipher.js:1]: Not 100% sure about the decision you took to capitalize here, it is not a class nor a constructor.

File: test/array-analysis.test.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+const analyze = require('../src/array-analysis');
+
+const array = [1, 8, 3, 4, 2, 6];
+const obj = analyze(array);
+
+test('it analyses an array and returns an object', () => {
+  expect(analyze([1, 8, 3, 4, 2, 6])).toStrictEqual({
```
Comment (MauricioRobayo) [inline test/array-analysis.test.js]: Maybe your intention was to use `obj` here? 🤔

Also curious about the decision you made to use `toStrictEqual` instead of `toEqual` 🤔.

File: test/array-analysis.test.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  expect(obj.min).toBe(1);
+});
+
+test('object returned has an max property', () => {
+  expect(obj.max).toBe(8);
+});
+
+test('object returned has an length property', () => {
+  expect(obj.length).toBe(6);
+});
```
Comment (MauricioRobayo) [inline test/array-analysis.test.js]: These tests might be redundant, you are already checking this with the first one. Maybe you can include some kind of other edge cases instead of double-checking here, for example: [1, null], [0], [1,1,1,1], [-Infinity, Infinity], etc. but there is no need for that in this project.

**Review by MauricioRobayo — APPROVED** [review]
I could not find any issue in this repository. You are doing great, really good code.

Status: __Approve__. 🚀

File: test/array-analysis.test.js (hunk 2 above)
Comment (Aaron-RN) [reply test/array-analysis.test.js]: Oh, I used toBe and it gave an error and jest suggested I used toStrictEqual.
